cluster-configuration-example/cluster-customize-configuration.yaml

```diff
@@ -0,0 +1,154 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+
+# If corresponding values aren't be set in the machine list, the default value will be filled in.
+DefaultMachineProperties:
+  user: username
+  password: password
+  sshport: port
+
+
+machineinfo:
+
+  NC24R:
+    mem: 224
+    gpu:
+    # type: gpu{type}
+      type: teslak80
+      count: 4
+    cpu:
+      vcore: 24
+    dataFolder: "/mnt"
+    # Note: Up to now, the only supported os version is Ubuntu16.04. Please do not change it here.
+    os: ubuntu16.04
+
+  D8SV3:
+    mem: 32
+    cpu:
+      vcore: 8
+    dataFolder: "/mnt"
+    # Note: Up to now, the only supported os version is Ubuntu16.04. Pls don't change it here.
+    os: ubuntu16.04
+
+
+
+machinelist:
+  # hostname, must be the same as it in the host network
+  infra-01:
+    # The nodename in k8s. To avoid complication, it is suggested to fill in the hostip for the nodename.
+    # If you are confident that DNS can resolve your named nodename correctly, you can specify a UNIQUE_HOST_NAME.
+    # hostip: the static IP of this node. We assume the IP of a host will not change.
+    # sshport: the ssh port of this node. If not specified, the port is 22.
+    # username/password: the user name and password in the OS. Make sure it is granted with sudo permission.
+    hostname: XXXX
```

Comment:
>hostname: XXXX [](start = 4, length = 14)

as discussed, please remove this field to avoid confusion. in the deployment, just fill it the same as IP

Reply:
Nodename has been removed. Hostname should be kept.